coalib/coala_json.py

```diff
@@ -45,11 +46,18 @@ def main():
                 results.append(bear)
         except BaseException as exception:  # pylint: disable=broad-except
             return get_exitcode(exception, log_printer)
+    elif args.show_capabilities:
+        local_bears, global_bears = get_filtered_bears(
+            args.filter_by_language, log_printer)
+        capabilities = filter_capabilities_by_languages(
+            local_bears, args.show_capabilities)
     else:
         results, exitcode, _ = run_coala(
             log_printer=log_printer, autoapply=False)
 
-    retval = {"bears": results} if args.show_bears else {"results": results}
+    retval_ = ({"capabilities Can detect/Can fix": capabilities}
```

Comment:
The current output is:
```json
{
  "capabilities Can detect/Can fix": {
    "python": [
      [
        ...
      ],
      [
        ...
      ]
    ]
  }
}
```
It would be better to make `python` a dict instead of a list. So we can have something like:
```json
{
  "capabilities": {
    "python": {
      "CAN_DETECT": [
        ...
      ],
      "CAN_FIX": [
        ...
      ]
    }
  }
}
```